client-course-schedulizer/src/components/reuseables/AddSectionPopover/AddSectionPopover.tsx

```diff
@@ -48,6 +57,9 @@ export const AddSectionPopover = () => {
   const isIntensiveSemester = semesterLength === SemesterLengthOption.IntensiveSemester;
   const isCustomSemester = semesterLength === SemesterLengthOption.CustomSemester;
 
+  const locationValue =
+    (values && `${values?.meeting.location.building} ${values?.meeting.location.roomNumber}`) || "";
```

Comment:
this results in the string `" "` when no `building` or `roomNumber` is specified (I think because `" "` returns true). I guess:
```ts
const locationValue = (values && (values.meeting.location.building || values.meeting.location.roomNumber)) ? `${values.meeting.location.building} ${values.meeting.location.roomNumber}` : ""
```
might work but looks clunky (maybe could also try using `trim()` with the current implementation)

Reply:
Good catch, thanks.

Reply:
As a note, I think
```ts
const locationValue =
    ((values && `${values?.meeting.location.building} ${values?.meeting.location.roomNumber}`) || "").trim();
```
Might be the simplest/best approach